spec/integration/lib/auth0/api/v2/api_resource_servers_spec.rb

```diff
@@ -98,6 +98,33 @@
     end
   end
 
+    describe '.patch_client', vcr: true do
+    it 'should raise an error with a missing client_id' do
```

Comment:
We can take out the integration specs that check for a raised error. The unit specs will cover just fine.